pyproject.toml

```diff
@@ -1,17 +1,109 @@
 [build-system]
+build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools<66",
+    "setuptools>=66,<68",
+    "setuptools_scm[toml]>=3.5.0",
     "wheel",
-    "setuptools_scm[tomp]>=3.5.0",
 ]
 
+[project]
+name = "catalystcoop.ferc_xbrl_extractor"
+description = "A tool for extracting data from FERC XBRL Filings."
+readme = {file = "README.rst", content-type = "text/x-rst"}
+authors = [
+    {name = "Catalyst Cooperative", email = "pudl@catalyst.coop"},
+    {name = "Zach Schira", email = "zach.schira@catalyst.coop"}
+]
+requires-python = ">=3.10,<3.12"
+dynamic = ["version"]
+license = {file = "LICENSE.txt"}
+dependencies = [
```

Comment:
A bunch of dependencies had stale upper bounds, so I increased them to match what we've got on PUDL. We should get the dependabot working on this repo, otherwise it'll end up constraining what versions PUDL can use, since PUDL depends on this repo.

Reply:
I'll see if I can get dependabot working